pkgs/top-level/all-packages.nix

```diff
@@ -9371,86 +9371,15 @@ in
   pachyderm = callPackage ../applications/networking/cluster/pachyderm { };
 
   php = php74;
-  phpPackages = php74Packages;
-
-  php72Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
-    php = php72;
-  });
-
-  php73Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
-    php = php73;
-  });
-
-  php74Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
-    php = php74;
-  });
-
-  phpPackages-unit = php74Packages-unit;
 
-  php72Packages-unit = recurseIntoAttrs (callPackage ./php-packages.nix {
```

Comment:
I very much approve inlining all these into the only derivations that use it, but we still need to add the attributes to `pkgs/top-level/aliases.nix` with a throw explaining they were inlined, and how to obtain a similarily-configured PHP.

Reply:
I've now made two fixup commits to add these to the aliases files, the commit ids are 9e51ba64d9e and 98b77d59ba0.

Me and @talyz can squash them in later :-)